lib/ex_doc/formatter/html/templates.ex

```diff
@@ -59,7 +59,8 @@ defmodule ExDoc.Formatter.HTML.Templates do
   def synopsis(nil), do: nil
   def synopsis(""),  do: ""
   def synopsis(doc) when is_bitstring(doc) do
-    String.split(doc, ~r/\n\s*\n/)
+    doc
+    |> String.split(~r/\n\s*\n/)
     |> hd
```

Comment:
I'm working seein the line bellow, if we should do `|> hd()`, i know hd has only one function with arity one, but in the unlinkely event of a having hd/2 in the future, this code can break, i guess. just a convention maybe that can be good to follow. what do you guys think?